### ISSUE TITLE
Warning: Declaration of CartEvent should be compatible with Cart::addItem

### ISSUE THREAD

**Issue by madc** [thelia-modules/LegacyProductAttributes#4]
Thelia v2.3.3
After installation, this issue pops up on top of the admin area:

```
Warning: Declaration of LegacyProductAttributes\EventListeners\CartAction::addItem(Thelia\Core\Event\Cart\CartEvent $event) should be compatible with Thelia\Action\Cart::addItem(Thelia\Core\Event\Cart\CartEvent $event, $eventName, Symfony\Component\EventDispatcher\EventDispatcherInterface $dispatcher) in /mnt/MeShared/Development/Web/htdocs/clients/shop.heidenspass.cc/local/modules/LegacyProductAttributes/EventListeners/CartAction.php on line 158
```

**Comment by madc** [comment]
Making these changes in `local/modules/LegacyProductAttributes/EventListeners/CartAction.php` fixes the warning. Not tested, if the module still works..

```
    /**
     * Manage adding an item to the cart when our legacy product attributes are used.
     *
     * @param CartEvent $event
     * @param $eventName
     * @param EventDispatcherInterface $dispatcher
     */
    public function addItem(CartEvent $event, $eventName, EventDispatcherInterface $dispatcher)
    {
        $this->getLegacyProductAttributes($event);

        // call the parent method, but using our redefined sub-methods
        parent::addItem($event);

        // prevent the parent event from adding the item
        $event->setNewness(false);
        $event->setAppend(false);
    }
```

**Comment by roadster31** [comment]
Fixed by #5
